QUARKUS-4221.md

```diff
@@ -6,10 +6,7 @@ PR: https://github.com/quarkus-qe/quarkus-test-suite/pull/2003
 ## Scope of the testing
 ### What to test
 
-The test coverage will be verified for environments: JVM for Baremetal.
-
- It won't be tested on Native because this support was requested by the Keycloak team, and they explicitly stated (in the comments of the Jira task) that they don't work with native executables,
-so it's not needed at the moment.
+The test coverage will be verified with JVM mode on Baremetal.
```

Comment:
You removed native mode, so now it only says JVM mode. _What to test_ should say that native mode will be tested too?